var/spack/repos/builtin/packages/mvapich2x/package.py

```diff
@@ -198,11 +199,6 @@ def construct_cflags(self):
             cflags = cflags + "-I/opt/xpmem/include"
         return cflags
 
-    def setup_build_environment(self, env):
-        # mvapich2 configure fails when F90 and F90FLAGS are set
```

Comment:
This should also probably be kept

Reply:
See above. This one seems also an evidence of the copy/paste from `mvapich2`.